services/web/server/src/simcore_service_webserver/application_settings_utils.py

```diff
@@ -215,6 +215,12 @@ def _set_if_disabled(field_name, section):
 
         _set_if_disabled("WEBSERVER_DB", db)
 
+    if section := cfg.get("rabbitmq"):
```

Comment:
we need to deprecate `config` asap ... it is adding unnecessary maintenance

Reply:
I can only but fully agree